cluster/manifests/z-kubenurse/service.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if and (eq .Cluster.ConfigItems.network_monitoring_enabled "true") (eq .Cluster.ConfigItems.network_monitoring_check_kubenurse_service "true")}}
```

Comment:
Since the Ingress needs the Service usually, we might need to create it when `network_monitoring_check_kubenurse_ingress == true` as well.